packages/edit-site/src/components/global-styles/font-library-modal/font-collection.js

```diff
@@ -153,34 +140,67 @@ function FontCollection( { id } ) {
 		setFontsToInstall( [] );
 	};
 
+	const handleFontDownload = async ( fontFamily ) => {
+		return Promise.all(
+			fontFamily.fontFace.map( async ( fontFace ) => {
+				if ( fontFace.src ) {
+					fontFace.file = await downloadFontFaceAsset( fontFace.src );
+				}
+			} )
+		).then( () => {
+			return fontFamily;
+		} );
+	};
+
 	const handleInstall = async () => {
-		const fontFamily = fontsToInstall[ 0 ];
+		let fontFamily = fontsToInstall[ 0 ];
+
+		//NOTE: This is only necessary while the collection potentially includes the depreciated 'downloadFromUrl' property.
+		if ( fontFamily?.fontFace ) {
+			fontFamily.fontFace = fontFamily.fontFace.map( ( fontFace ) => {
+				fontFace.src = fontFace.downloadFromUrl || fontFace.src;
+				delete fontFace.downloadFromUrl;
+				return fontFace;
+			} );
+		}
+		////////////
```

Comment:
Is this to indicate what chunk of code can be removed once `downloadFromURL` is deprecated? I think it's clear enough without it.